packages/itwin/map-layers/package.json

```diff
@@ -115,6 +115,9 @@
     "@itwin/core-frontend": "^4.1.2",
     "@itwin/core-react": "^4.0.0",
     "@itwin/imodel-components-react": "^4.0.0",
+    "@itwin/itwinui-variables": "^2.0.0",
+    "@itwin/itwinui-react": "^2.11.2",
```

Comment:
itwinui should not be a peer

Reply:
If this package consumes it as a direct dep, then it ought to be a peer to communicate to the package users that it's required.
i guess it is *assumed* this package lives in the larger itwinjs bubble so the dependency on itwinui stuff is implied?

Reply:
itwinui should be a direct dep imo

Reply:
itwinui should be direct;
need to review if map-layers-formats can be a direct dep. i think that might actually need to be a peer if it has statics and is not pure